app/models/dashboard.rb

```diff
@@ -0,0 +1,11 @@
+class Dashboard
```

Comment:
This would be better as a Solicitors model so we could do:

```
Solicitor.all
```
which would return an array of Solicitor models